coreneuron/sim/solve_core.cpp

```diff
@@ -37,17 +39,9 @@ static void triang(NrnThread* _nt) {
     double* vec_rhs = &(VEC_RHS(0));
     int* parent_index = _nt->_v_parent_index;
 
-#if defined(_OPENACC)
-    int stream_id = _nt->stream_id;
-#endif
-    /** @todo: just for benchmarking, otherwise produces wrong results */
-    // clang-format off
-
-    #pragma acc parallel loop seq present(      \
-        vec_a[0:i3], vec_b[0:i3], vec_d[0:i3],  \
-        vec_rhs[0:i3], parent_index[0:i3])      \
-        async(stream_id) if (_nt->compute_gpu)
-    // clang-format on
+    nrn_pragma_acc(parallel loop seq present(
+        vec_a [0:i3], vec_b [0:i3], vec_d [0:i3], vec_rhs [0:i3], parent_index [0:i3])
+                       async(_nt->stream_id) if (_nt->compute_gpu))
     for (int i = i3 - 1; i >= i2; --i) {
```

Comment:
just a note: the weird clang-format still hurts my eyes but this is manageable / simple 😃

Reply:
🤷 this one was just below threshold for me. It's nice not to have _too many_ `// clang-format on|off` comments punctuating things.